core/state/database.go

```diff
@@ -194,6 +197,9 @@ func (db *cachingDB) OpenTrie(root common.Hash) (Trie, error) {
 
 // OpenStorageTrie opens the storage trie of an account.
 func (db *cachingDB) OpenStorageTrie(stateRoot common.Hash, address common.Address, root common.Hash) (Trie, error) {
+	if db.triedb.IsZkTree() {
+		return trie.NewZkMerkleStateTrie(root, db.triedb)
```

Comment:
I remember that you would change this to `NewZkMerkleStateTree`, right?

Reply:
We decided to replace *tree with *trie